java/src/main/java/org/wildfly/openssl/ByteBufferUtils.java

```diff
@@ -54,6 +55,9 @@ public static ByteBuffer expand(ByteBuffer in) {
      * need for expansion
      */
     public static ByteBuffer expand(ByteBuffer in, int newSize) {
+    	Assert.checkNotNullParam("in", in);
+    	Assert.checkMinimumParameter("newSize", 0, newSize);
+    	
```

Comment:
You may run into issues with our checkstyle validation here on these lines.
It looks like the the indentations on line `58` and `59` have a mix of Tabs and Spaces. Could you please switch these to all spaces (for 2 indents it would be 8 spaces in total).
And on line `60` you have a trailing indent, so if you could remove the extra spaces/tabs on that line, that would be great!